ci/infra/testrunner/requirements.txt

```diff
@@ -1,5 +1,6 @@
 timeout-decorator
 requests
 pyhcl
-pytest
+pytest==5.1.0
```

Comment:
why this specific version?

Reply:
> why this specific version?

If version specific is better than blind version. If someone already had python virtual env a long time back, It would not use without upgrading the packages.  If we add a version, all the package version will be same in all dev machines. Plus, some pytest plugins requires high version of pytest.